crates/swc_ecma_transforms_compat/tests/es2018_object_rest_spread.rs

```diff
@@ -523,7 +522,7 @@ const {
   },
   c = ({ ...d } = {}),
 } = {};
-a()
+a({})
```

Comment:
I think this is a wrong test case. @kdy1 
`a()` should throw  `Cannot destructure undefined`

Reply:
I tested on chrome and I think you are right